### PR TITLE
Training models on all full datasets

### PR DESCRIPTION
**Pull request by nthain**
* Rebuilds eval_tool into model_tool to make training multiple models easier
* Trains models on wikipedia datasets alone, augmented with debias data, and augmented with random data

NB: This branch contains jetpack's commits as they have not yet been merged into master but I will review them seperately in his PR.

### CODE REVIEW

**Comment by jetpack** [comment]
Could you add a description to the README of what all the different dataset files are?

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ These notebooks are written for Python 2.7. To run them:
 ```
 pip install -r requirements.txt
 ```
-2. (optional: to skip training) Download the latest [model](https://storage.googleapis.com/unintended-ml-bias-analysis/models/wiki_tox_labels_v1_model.h5) and [tokenizer](https://storage.googleapis.com/unintended-ml-bias-analysis/models/wiki_tox_labels_v1_tokenizer.pkl) to the `models/` subdirectory.
+2. (optional: to skip training) Download the latest [model](https://storage.googleapis.com/unintended-ml-bias-analysis/models/[MODEL_NAME]_model.h5) and [tokenizer](https://storage.googleapis.com/unintended-ml-bias-analysis/models/[MODEL_NAME]_tokenizer.pkl) to the `models/` subdirectory.
```
Comment (jetpack) [inline README.md]: Is there a list of the valid `MODEL_NAME` values somewhere?

**Comment by nthain** [comment]
Also should remove eval_tool.py

**Comment by nthain** [comment]
Change hyperparameters to the newest version:

MAX_SEQUENCE_LENGTH = 250
MAX_NUM_WORDS = 15000
EMBEDDING_DIM = 100
EMBEDDING_TRAINABLE = False
LEARNING_RATE = 0.00005

BATCH_SIZE = 128
EPOCHS = 20
DROPOUT_RATE = 0.3
CNN_FILTER_SIZES = [128, 128, 128]
CNN_KERNEL_SIZES = [5,5,5]
CNN_POOLING_SIZES = [5, 5, 40]

File: README.md (hunk 1 above)
Comment (nthain) [reply README.md]: Not yet. We've changed to training many models so will need to revisit this later.

**Comment by nthain** [comment]
> We should pull out the code in the "Model bias analysis" notebook into a library so we don't need to duplicate it for each notebook.

Agreed. Thanks for doing that jetpack! :-)

**Comment by nthain** [comment]
Responded to comments, PTAL.

**Pull request merged**